### PR TITLE
ONNX export: Support equal length splits

### PR DESCRIPTION
**Pull request by vandanavk**
## Description ##
Fixes https://github.com/apache/incubator-mxnet/issues/13061

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- Support for exporting and importing split operator when the split are of equal length

### CODE REVIEW

**Comment by vandanavk** [comment]
@mxnet-label-bot add [pr-awaiting-review, ONNX]

@Roshrini for review

**Comment by vandanavk** [comment]
Thanks for reviewing @ChaiBapchya. The tests listed in test_cases.py are [ONNX operator unit test cases ](https://github.com/onnx/onnx/blob/master/onnx/backend/test/case/node/split.py). These test operator import/export. These are different from the operator test cases written in tests/python/unittest/test_operator.py of MXNet.

File: python/mxnet/contrib/onnx/mx2onnx/export_onnx.py [hunk 1]
```diff
@@ -262,17 +262,18 @@ def create_onnx_graph_proto(self, sym, params, in_shape, in_type, verbose=False)
...
                         onnx_processed_nodes.append(converted_node)
-                        node_name = converted_node.name if converted_node.name else converted_node.output[0]
-                        if node_name in graph_outputs:
-                            onnx_processed_outputs.append(
-                                make_tensor_value_info(
-                                    name=node_name,
-                                    elem_type=in_type,
-                                    shape=graph_outputs[node_name]
+                        node_names = list(converted_node.output)
+                        for nodename in node_names:
```
Comment (Roshrini) [inline python/mxnet/contrib/onnx/mx2onnx/export_onnx.py]: Can you add code comments here?

Comment (vandanavk) [reply python/mxnet/contrib/onnx/mx2onnx/export_onnx.py]: Done

**Pull request merged**